src/main/java/chatbot/Storage.java

```diff
@@ -0,0 +1,96 @@
+package chatbot;
+
+import chatbot.task.Task;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
Would it make things clearer to determine the relevant used dependencies if the different import statements where separated?
```suggestion
import java.io.IOException;

import java.util.ArrayList;
import java.util.Scanner;
```